github/README.md

```diff
@@ -27,43 +28,59 @@ Creating an issue.
 ```
 /nc github_create_issue <repo> <title> <body>
 ```
-![GitHub create_issue command](https://raw.githubusercontent.com/nimbella/command-sets/master/github/screenshots/create_issue.PNG)
+![GitHub create_issue command](https://raw.githubusercontent.com/nimbella/command-sets/master/github/screenshots/create_issue.png)
 
 Closing an issue.
 ```
 /nc github_close_issue <repo> <issue_number>
 ```
-![GitHub close_issue command](https://raw.githubusercontent.com/nimbella/command-sets/master/github/screenshots/close_issue.PNG)
+![GitHub close_issue command](https://raw.githubusercontent.com/nimbella/command-sets/master/github/screenshots/close_issue.png)
 
 Reopening an issue
 ```
 /nc github_reopen_issue <repo> <issue_number>
 ```
-![GitHub reopen_issue command](https://raw.githubusercontent.com/nimbella/command-sets/master/github/screenshots/reopen_issue.PNG)
+![GitHub reopen_issue command](https://raw.githubusercontent.com/nimbella/command-sets/master/github/screenshots/reopen_issue.png)
 
 Finding a pull request by date.
 ```
 /nc github_find_pr <repo> <date>
 ```
-![GitHub find_pr command](https://raw.githubusercontent.com/nimbella/command-sets/master/github/screenshots/find_pr.PNG)
+![GitHub find_pr command](https://raw.githubusercontent.com/nimbella/command-sets/master/github/screenshots/find_pr.png)
 
 Labeling an issue or pull request.
 ```
 /nc github_label <repo> <number> <labels>
 ```
-![GitHub label command](https://raw.githubusercontent.com/nimbella/command-sets/master/github/screenshots/label.PNG)
+![GitHub label command](https://raw.githubusercontent.com/nimbella/command-sets/master/github/screenshots/label.png)
 
 Getting repository statistics
 ```
 /nc github_stats <repo>
 ```
-![GitHub stats command](https://raw.githubusercontent.com/nimbella/command-sets/master/github/screenshots/stats.PNG)
+![GitHub stats command](https://raw.githubusercontent.com/nimbella/command-sets/master/github/screenshots/stats.png)
 
 Viewing recent pull requests
 ```
 /nc github_view_pr <repo>
 ```
-![GitHub view_pr command](https://raw.githubusercontent.com/nimbella/command-sets/master/github/screenshots/view_pr.PNG)
+![GitHub view_pr command](https://raw.githubusercontent.com/nimbella/command-sets/master/github/screenshots/view_pr.png)
+
+Finding repositories, commits, code, issues, pull requests, users and topics using keywords
+```
+/nc github_list <entity> <keywords> [<repository>] [<language>]

```

Comment:
```suggestion
/nc github_list <entity> <keywords> [-r <repository>] [-l <language>]
```
This might be more clear to the reader.

Reply:
Which one is better `-r <repository>` or `[-r <repository>]`?

Reply:
[-r ...] isn't the whole thing optional?

Reply:
Yes. It's just that `[-r ...]` doesn't look great to me so I wondered. 😁